fedn/fedn/clients/reducer/statestore/mongoreducerstatestore.py

```diff
@@ -191,7 +191,11 @@ def get_model_info(self):
                 return None
         except (KeyError, IndexError):
             return None
-    
+
+    def get_events(self):
+        ret = self.control.status.find({})
```

Comment:
This will become  a big collection quickly. Filter on X latest?

Reply:
It will have to do for now. In next iteration we may introduce memcached reads and filtering to the api